src/app/component/SearchUser.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState, Fragment } from "react";
+import { Box, TextField, Avatar } from "@material-ui/core";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import { makeStyles } from "@material-ui/core/styles";
+import _ from "lodash";
+
+import { Users } from "../model";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    paddingBottom: theme.spacing(2),
+  },
+  avatar: {
+    marginRight: theme.spacing(1),
+  },
+  textField: {
+    paddingRight: "9px!important",
+  },
+}));
+
+const SearchUser = ({ value, setValue }) => {
+  const classes = useStyles();
+
+  const [users, setUsers] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [open, setOpen] = useState(false);
+
+  const handleInputChange = async (event, value, reason) => {
+    if (value.length > 1) {
+      setLoading(true);
+
+      const users = await Users.usersMatchingLabel(value, 10);
```

Comment:
Can we use a named variable for the magic number (10) ?

Reply:
Will do